.github/workflows/build.yml

```
@@ -180,10 +185,10 @@ jobs:
       - name: Save unit tests coverage in cache 🥡
         uses: actions/cache/save@v4
         id: cache-unit-tests-coverage
-        if: ${{ hashFiles('tests/unit/coverage/lcov.info') }}
+        if: ${{ hashFiles('tests/unit/coverage/lcov.info') != '' }}
         with:
-          path: tests/unit/coverage
-          key: ${{ runner.os }}-tests-unit-coverage-v3-${{ env.BRANCH_NAME }}-${{hashFiles('tests/unit/coverage/lcov.info')}}
+          path: ${{ env.UNIT_TESTS_COVERAGE_REPORTS_PATH }}
+          key: ${{ env.UNIT_TESTS_COVERAGE_BASE_CACHE_KEY }}-${{ steps.safe-branch-name.outputs.safe-branch-name }}-${{ hashFiles('tests/unit/coverage/lcov.info') }}
```

actionlint (1.7.7)

property "safe-branch-name" is not defined in object type {}

(expression)